stablehlo/dialect/TypeInference.cpp

```diff
@@ -2736,20 +2724,22 @@ LogicalResult inferSliceOp(std::optional<Location> location, Value operand,
                                  operandSizeOrBound, " in dimension ", i);
```

Comment:
missing // SliceOp_C3
`limit_indices[d] <= dim(operand, d) for all dimension d.`